MobiusCore/Source/MobiusController.swift

```diff
@@ -33,36 +33,67 @@ public final class MobiusController<Model, Event, Effect> {
     private let state: State
 
     /// A Boolean indicating whether the MobiusLoop is running or not.
-    ///
-    /// May not be called directly from the update function or an effect handler running on the controller’s loop queue.
     public var running: Bool {
         return state.running
     }
 
+    /// See `Mobius.Builder.makeController` for documentation
     init(
         builder: Mobius.Builder<Model, Event, Effect>,
         initialModel: Model,
-        loopQueue: DispatchQueue,
+        loopQueue loopTargetQueue: DispatchQueue,
         viewQueue: DispatchQueue
     ) {
-        let actualLoopQueue = DispatchQueue(label: "MobiusController \(Model.self)", target: loopQueue)
+        // The internal loopQueue is a serial queue targeting the provided queue, so that targeting a concurrent queue
+        // doesn’t result in concurrent work on the underlying MobiusLoop.
```

Comment:
Is 'targeting' a synchronous operation? What I mean is, is there a risk of the following:

1. Event A arrives and is placed on `loopQueue`
1. Event B arrives and is placed on `loopQueue`
1. Event A starts processing. 
1. Event B starts processing concurrently with A.

Reply:
When a serial dispatch queue `Q` (which is the default type) feeds into a concurrent dispatch queue `P`, blocks submitted to `Q` are guaranteed to be executed serially with respect to each other, although they may be concurrent with other blocks submitted to `P`. So step 4 in your scenario can’t happen.

Reply:
Great!